sdcm/tester.py

```diff
@@ -931,32 +988,35 @@ def get_cluster_k8s_gce_minikube(self):
         self.k8s_cluster.docker_pull(f"{self.params.get('docker_image')}:{self.params.get('scylla_version')}")
 
-        self.db_cluster = \
-            minikube.MinikubeScyllaPodCluster(k8s_cluster=self.k8s_cluster,
-                                              scylla_cluster_name=self.params.get("k8s_scylla_cluster_name"),
-                                              user_prefix=self.params.get("user_prefix"),
-                                              n_nodes=self.params.get("n_db_nodes"),
-                                              params=self.params)
+        self.db_cluster = mini_k8s.RemoteMinimalScyllaPodCluster(
+            k8s_cluster=self.k8s_cluster,
+            scylla_cluster_name=self.params.get("k8s_scylla_cluster_name"),
+            user_prefix=self.params.get("user_prefix"),
+            n_nodes=self.params.get("n_db_nodes"),
+            params=self.params,
+        )
 
         self.log.debug("Update startup script with iptables rules")
         startup_script = "\n".join((Setup.get_startup_script(), *self.db_cluster.nodes_iptables_redirect_rules(),))
         Setup.get_startup_script = lambda: startup_script
 
-        self.loaders = LoaderSetGCE(gce_image=self.params.get("gce_image"),
-                                    gce_image_type=self.params.get("gce_root_disk_type_loader"),
-                                    gce_image_size=None,
-                                    gce_network=self.params.get("gce_network"),
-                                    service=services[:1],
-                                    credentials=self.credentials,
-                                    gce_instance_type=self.params.get("gce_instance_type_loader"),
-                                    gce_n_local_ssd=self.params.get("gce_n_local_ssd_disk_loader"),
-                                    gce_image_username=self.params.get("gce_image_username"),
-                                    user_prefix=self.params.get("user_prefix"),
-                                    n_nodes=self.params.get('n_loaders'),
-                                    params=self.params,
-                                    gce_datacenter=gce_datacenter)
+        self.loaders = LoaderSetGCE(
+            gce_image=self.params.get("gce_image"),
+            gce_image_type=self.params.get("gce_root_disk_type_loader"),
+            gce_image_size=None,
+            gce_network=self.params.get("gce_network"),
+            service=services[:1],
+            credentials=self.credentials,
+            gce_instance_type=self.params.get("gce_instance_type_loader"),
+            gce_n_local_ssd=self.params.get("gce_n_local_ssd_disk_loader"),
+            gce_image_username=self.params.get("gce_image_username"),
+            user_prefix=self.params.get("user_prefix"),
+            n_nodes=self.params.get('n_loaders'),
+            params=self.params,
+            gce_datacenter=gce_datacenter,
+        )
```

Comment:
In general it looks nicer, yes. But this change is already huge and such changes only complicate review.

Reply:
Let's keep it.

Reply:
Having "+831" ne lines "−368" removed ones we must reduce it if possible.

Again, it is not ok to bloat already huge change even more. 
Please, move such "nice" enhancements into a separate PR.

Reply:
done